ui/lib/core/addon/components/secrets-engine-mount-config.ts

```diff
@@ -36,6 +36,7 @@ export default class SecretsEngineMountConfig extends Component<Args> {
       { label: 'Secret Engine Type', value: model.engineType },
       { label: 'Path', value: model.path },
       { label: 'Accessor', value: model.accessor },
+      { label: 'Running Plugin Version', value: model.runningPluginVersion },
```

Comment:
According to our capitalization policy (confirmed in the Figma file "Capitalization"), we use Title Case for headings and titles, but Sentence case for attributes and labels. I initially used `Running Plugin Version` to match the existing labels on this form, but technically, it should be sentence-cased. If you feel so inclined, could you update the rest of the Labels on this model to match this pattern? Note: because of openAPI the only time we don't use sentence-case is with Default Lease TTL or Max Lease TTL. 
```suggestion
      { label: 'Running plugin version', value: model.runningPluginVersion },
```